src/main/java/org/folio/rest/impl/ItemStorageAPI.java

```diff
@@ -56,6 +56,13 @@ public void postItemStorageItems(
       RoutingContext routingContext, Map<String, String> okapiHeaders,
       Handler<AsyncResult<Response>> asyncResultHandler,
       Context vertxContext) {
+    //if the initial status is "Available", set the last updated date of that 
+    //status to be the creation date of the item
+    
+    if (entity.getStatus().getName().toString().equals("Available")) {
```

Comment:
Why is this date only set when the status is available?

Reply:
Because the bug is written in such a way that it implies this is only desirable when the initial status is available.  As I said above, I'm not entirely sure this check is necessary, from what I can see, newly-created items can *only* have a status of available.

Reply:
Please can you check with the author of the JIRA issue if that check is needed.

I believe it is possible to create items in other statuses, I think available is the default.

Reply:
Yes, I agree that we should set the date for all the statuses.

Reply:
Charlotte confirms this.  I'll remove the check.